### DISCUSSION TITLE
Is it possible run IPED in a cluster?

### DISCUSSION THREAD

**Discussion by Leanke-cwb** [sepinf-inc/IPED#227] [Q&A] [answered]
I'm trying to run the iPad on Hadoop so that it breaks the cloud packages and processes the parts of the file on the nodes to improve the processing time, is that possible?

**Comment by lfcnassif** [comment, marked as answer]
Using the command line interface, you can build a manager service to process different evidences in different processing nodes, but breaking processing of the same evidence in different nodes currently is not supported. It is desired, but not a high priority of the project for now.

**Comment by lfcnassif** [comment]
please reopen if you still has any questions.
